src/commands/targets.ts

```diff
@@ -0,0 +1,17 @@
+import { Argv, CommandBuilder } from 'yargs';
+import { getConfiguration } from '../config';
+import { getAllTargetNames, getTargetId } from '../targets';
+
+export const builder: CommandBuilder = (yargs: Argv) => {
+  return yargs;
+};
+
+export function handler(): any {
+  const definedTargets = getConfiguration().targets || [];
+  const possibleTargetNames = new Set(getAllTargetNames());
+  const allowedTargetNames = definedTargets
+    .filter(target => target.name && possibleTargetNames.has(target.name))
+    .map(getTargetId);
+
+  console.log(JSON.stringify(allowedTargetNames));
```

Comment:
wdyt about adding indent or using `formatJson` (craft's utility function to print an indented JSON) here?

Reply:
Oh, yeah that sounds lovely.